text/0174-metrics-api.md

```diff
@@ -107,6 +122,24 @@ Returns the total number of index for the Meilisearch instance.
 meilisearch_index_count :numberOfIndex
 ````
 
+#### 3.2.7. `meilisearch_nb_tasks`
+
+Returns the total number of index for the Meilisearch instance.
```

Comment:
hey, small typo found passing by reading more on meilisearch specs repo
seems a copy past of the above metric name desc